onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -45,16 +45,200 @@ TensorrtExecutionProvider::TensorrtExecutionProvider()
 
 TensorrtExecutionProvider::~TensorrtExecutionProvider() {}
 
+std::unique_ptr<IndexedSubGraph> TensorrtExecutionProvider::GetSubGraph(SubGraph_t graph_nodes_index, int& kernels_index, const onnxruntime::GraphViewer& graph) const {
```

Comment:
some comments describing the algorithm and motivation for the new approach for implementing GetCapability would be nice.
maybe add some comments for GetSubGraph and GetSupportedList

Reply:
Add more comments for those two new functions